sdk/core/core-auth/rollup.base.config.js

```diff
@@ -96,9 +96,6 @@ export function browserConfig(test = false, production = false) {
     // the "sideEffects" field in package.json.  Since our package.json sets "sideEffects=false", this also
     // applies to test code, which causes all tests to be removed by tree-shaking.
     baseConfig.treeshake = false;
-  } else if (production) {
```

Comment:
If we are not dealing with the "production" case, then maybe update `browserConfig()` to not take the `production` boolean input in the first place?

Reply:
Good catch. Will remove the unused argument.